libs/components/forms/src/lib/modules/input-box/input-box.component.ts

```diff
@@ -164,6 +164,17 @@ export class SkyInputBoxComponent
 
   public hostIconsInsetLeftTemplate: TemplateRef<unknown> | undefined;
 
+  public set hostHintText(value: string | undefined) {
+    this.#_hostHintText = value;
+    this.#changeRef.markForCheck();
+  }
+
+  public get hostHintText(): string | undefined {
+    return this.#_hostHintText;
```

Comment:
We have to use getter/setters for inputs (until we get `signal` support), but they're not ideal for template saturation because the `get` is called for every change detection cycle (put a `console.log` in the `get` method to see how many times they get called when serving the playground SPA). 

Instead, I'd use a static protected property and trigger change detection when it changes.

```
protected hostHintText: string | undefined;

public setHostHintText(value: string): void {
  this.hostHintText = value;
  this.#changeRef.markForCheck();
}
```

Reply:
Good call. Fixed.